src/main/java/com/google/alpollo/model/SongInfo.java

```diff
@@ -0,0 +1,38 @@
+package com.google.alpollo.model;
+
+import java.util.List;
+import com.google.alpollo.Song;
+import com.google.alpollo.model.SongEntity;
+import com.googlecode.objectify.annotation.Index;
+import com.googlecode.objectify.annotation.Entity;
+
+/**
+ * Class used for storing data related to the Natural Language API analysis.
+ * Extending the Song class, it provides all the info related to a specific song.
+ * 
+ * We extend from Song because we do not always need the entire song data. Sasha mostly
```

Comment:
Do not mention names in the javadoc, and think whether what you write can be understood by someone with little to no context.